### PR TITLE
chore: rename mysql topology to semisync

### PR DESCRIPTION
**Pull request by xuriwuyun**

### CODE REVIEW

**Review by weicao — DISMISSED** [review]

File: addons-cluster/mysql/values.schema.json [hunk 1]
```diff
@@ -75,10 +75,10 @@
       "default": "server",
       "description": "Topology type of the server",
       "enum": [
-        "server",
-        "server-with-proxy",
+        "semisync",
+        "semisync-with-proxysql",
```
Comment (weicao) [inline addons-cluster/mysql/values.schema.json:79]: semisync-proxysql is better

**Review by Y-Rookie — APPROVED** [review]

**Review by shanshanying — APPROVED** [review]

**Review by kubeJocker — APPROVED** [review]

**Comment by kubeJocker** [comment]
@xuriwuyun 
please check mysql cluster chart for the usage of topology

**Pull request merged**